app/jenkins-for-jira-ui/src/common/analytics/analytics-events.ts

```diff
@@ -38,12 +38,11 @@ export enum AnalyticsTrackEventsEnum {
 	ConnectedJenkinsServerSuccessName = 'connectedJenkinsServerSuccess',
 	ConnectedJenkinsServerErrorName = 'connectedJenkinsServerError',
 	UpdatedServerSuccessName = 'updatedServerSuccessName',
-	UpdatedServerErrorName = 'updatedServerErrorName'
-}
-
-export enum AnalyticsOperationalEventsEnum {
+	UpdatedServerErrorName = 'updatedServerErrorName',
 	GetServerSuccessManageConnectionName = 'getServerManageSuccessConnection',
 	GetServerErrorManageConnectionName = 'getServerErrorManageConnection',
 	DisconnectServerSuccessServerManageConnectionName = 'disconnectedServerSuccessManageConnection',
 	DisconnectServerErrorManageConnectionName = 'disconnectServerErrorManageConnection'
 }
+
+export enum AnalyticsOperationalEventsEnum {}
```

Comment:
Did a bit more reading on operational events and determined these fit better into the track category.